File: MODUL.5/PRAK501_2310817120010_NurHikmah.c
#include <stdio.h>

// Buatlah Function Disini
int MaxBilangan(int a, int b, int c, int d)
{
    int maksimal = a;
    if (b > maksimal)
    {
        maksimal = b;
    }
    if (c > maksimal)
    {
        maksimal = c;
    }
    if (d > maksimal)
    {
        maksimal = d;
    }
    return maksimal;
}

int main()
{
    int a, b, c, d;
    scanf("%d %d %d %d", &a, &b, &c, &d);
    int hasil = MaxBilangan(a, b, c, d);
    printf("%d", hasil);

    return 0;
}

File: MODUL.5/PRAK501_2310817120010_NurHikmah.py
def max_bilangan(a, b, c, d):
    maksimal = a
    if b > maksimal:
        maksimal = b
    if c > maksimal:
        maksimal = c
    if d > maksimal:
        maksimal = d
    return maksimal

a, b, c, d = map(int, input().split())
hasil = max_bilangan(a, b, c, d)
print(hasil)
